core/derive/src/tests/packed_layout.rs

```diff
@@ -21,8 +21,7 @@ fn unit_struct_works() {
             struct UnitStruct;
         }
         expands to {
-            #[allow(non_upper_case_globals)]
-            const _DERIVE_ink_core_storage2_traits_PackedLayout_FOR_UnitStruct: () = {
+            const _: () = {
```

Comment:
Why do you get rid of all these names with informational value?

Reply:
because these names do not really bear informational value and it is a feature that they can no longer potentially clash with other names.